### DISCUSSION TITLE
Need a help related to gitgraph

### DISCUSSION THREAD

**Discussion by snehabihani** [Blazor-Diagrams/Blazor.Diagrams#208] [Q&A] [answered]
https://www.nicoespeon.com/gitgraph.js/stories/?path=/story/gitgraph-js-1-basic-usage--default

Can we create something like this
I need to show merging also
I need to create something this
![tmp_1657125693089](https://user-images.githubusercontent.com/54053243/177601400-197eb599-cf54-424f-949b-01bd75471e2d.jpg)

**Comment by zHaytam** [comment, marked as answer]
Hello!
Judging from the storybook you mentioned, I would say this library is too overkill for something that simple.
